src/en/charms-bundles.md

```diff
@@ -44,6 +50,12 @@ running the following command:
 sudo apt-get install juju-quickstart
 ```
 
+E.g this bundle https://jujucharms.com/u/jean-deruelle/mobicents-restcomm-mysql-bundle on top of the page  you will find command line command you can use to deploy this bundle. 
```

Comment:
I'd suggest going with something simpler like:

https://jujucharms.com/turbo-lamp

and maybe just stick to the link and the command vs the 'top of the page' part.